asmcli/components/ca/managed-cas.sh

```diff
@@ -84,7 +84,6 @@ x_wait_for_enabling_workload_certificates() {
   local GKEHUB_API; GKEHUB_API="$1"
   local FLEET_ID; FLEET_ID="$2"
 
-  info "Waiting for the workload certificates enablement. This may take up to 20 minutes ..."
```

Comment:
This statement should not be deleted; it is useful to inform a user the long wait ahead.

Reply:
moved the statement below